src/query/tsdb/remote/codecs.go

```diff
@@ -192,23 +138,62 @@ func encodeTagMatchers(modelMatchers models.Matchers) (*rpc.TagMatchers, error)
 	}, nil
 }
 
+func encodeFanoutOption(opt storage.FanoutOption) (rpc.FanoutOption, error) {
+	switch opt {
+	case storage.FanoutDefault:
+		return rpc.FanoutOption_DEFAULT_OPTION, nil
+	case storage.FanoutForceDisable:
+		return rpc.FanoutOption_FORCE_DISABLED, nil
+	case storage.FanoutForceEnable:
+		return rpc.FanoutOption_FORCE_ENABLED, nil
+	}
+
+	return rpc.FanoutOption_FORCE_ENABLED,
+		fmt.Errorf("Unknown fanout option for proto encoding: %v\n", opt)
```

Comment:
Can `return 0, errorWhatever` here rather than needing to explicitly give a value, it can do the implicit conversion without complaining I believe.

Also for the error, small nit on removing capitalization from the first error, to keep the consistency of all log messages and errors all beginning with lowercase. Also probably don't need the `\n` at the end of the Errorf since not printing directly yeah?